quarkchain/evm/messages.py

```diff
@@ -267,8 +267,8 @@ def apply_transaction_message(
     state.logs = []
     if is_cross_shard:
         if (
-            state.qkc_config.XSHARD_ADD_RECIEPT_TIMESTAMP is not None
-            and state.timestamp >= state.qkc_config.XSHARD_ADD_RECIEPT_TIMESTAMP
+            state.qkc_config.ENABLE_EVM_TIMESTAMP is None
```

Comment:
Why not update devnet config to support that?  Because without EVM, the receipt will always succeed?  In addition, we do want to test receipt and evm together in devnet, right?

Reply:
yes we want to test xshard receipt and evm in devenet. with this change, if `ENABLE_EVM_TIMESTAMP` is not specified, the code will 1) apply xshard deposit as tx; 2) add corresponding receipt; 3) and of course, enable EVM. all of those are expected